utilities/persistent_cache/persistent_cache_test.cc

```diff
@@ -48,9 +48,6 @@ static void RemoveDirectory(const std::string& folder) {
 
   // cleanup files with the patter :digi:.rc
   for (auto file : files) {
-    if (file == "." || file == "..") {
-      continue;
-    }
     status = Env::Default()->DeleteFile(folder + "/" + file);
     assert(status.ok());
   }
```

Comment:
Can this entire routine be deleted and replaced with DestroyDr (from file_util.h)?

Reply:
Done.